application/helpers/artifact_helper.py

```diff
@@ -0,0 +1,8 @@
+"""Defines helper methods used with artifacts"""
+from flask import current_app
+
+
+def build_url(file_url):
+    """ Schema: fileserver/id_filename """
+    return current_app.config["FILE_SERVER"] + \
+        "/" + file_url
```

Comment:
Really not a fan of the name "helper" that could basically be anything and nobody knows where to use it and what to put here.

I think the reason you extracted it from the schema is because you wanted to use the method outside the Schemaclass. But it never was a private method to begin with. And it doesn't need to be an instance method either. If you just change it to a class Method you can call it via: `ArtifactSchema.build_url()` which I think is sufficient and we don't need an extra folder + module for one method.

Reply:
This is now a static method in artifact schema.